src/controllers/accountController.ts

```diff
@@ -0,0 +1,64 @@
+import { IAddress } from "../interface";
```

Comment:
Since the methods `pubkey()` and `bech32()` are deprecated in the `Address` class, I think we can create locally a new interface that has `toBech32()` and `getPublicKey()`, given the fact that everybody will use that class.